requirements.txt

```diff
@@ -5,5 +5,5 @@
 
 pyyaml>=3.10
 aiohttp>=3.7.4
-setuptools>=39.0
+setuptools>=42.0
```

Comment:
Leave this alone, RHEL8.6 highest version is `python3-setuptools-39.2.0-6.el8.noarch.rpm`